tensilelite/Tensile/cmake/TensileConfig.cmake

```diff
@@ -275,7 +275,7 @@ function(TensileCreateExtOpLibraries OutputFolder ArchStr)
     COMMAND ${CMAKE_COMMAND} -E rm -rf ${build_tmp_dir}
     COMMAND ${CMAKE_COMMAND} -E make_directory ${build_tmp_dir}
     COMMAND ${CMAKE_COMMAND} -E make_directory ${OutputFolder}
-    COMMAND bash "${script}" "\"${Archs}\"" "${build_tmp_dir}" "${VIRTUALENV_HOME_DIR}" "${Tensile_BUILD_ID}"
+    COMMAND PYTHONPATH=$ENV{PYTHONPATH} bash "${script}" "\"${Archs}\"" "${build_tmp_dir}" "${VIRTUALENV_HOME_DIR}" "${Tensile_BUILD_ID}"
```

Comment:
Ditto, set environment variables in command execution by prefixing with `${CMAKE_COMMAND} -E env ... --` (note that trailing double dash separates the remainder of the command line.

Reply:
Fixed